docs/src/components/pricing/PricingTable.tsx

```diff
@@ -95,58 +94,62 @@ export function PlanPrice(props: PlanPriceProps) {
     return (
       <div>
         <Box sx={{ display: 'flex', alignItems: 'center', mt: 1, mb: 2 }}>
-          <Typography
-            variant="body2"
-            fontWeight="bold"
-            color="error.500"
-            sx={{
-              borderRadius: 0.5,
-              bgcolor: (theme) => (theme.palette.mode === 'dark' ? 'error.900' : 'error.100'),
-              textDecoration: 'line-through',
-              p: '4px',
-            }}
-          >
-            $249
-          </Typography>
           <Box sx={{ width: 10 }} />
           <Typography variant="h4" component="div" fontWeight="bold" color="primary.main">
-            $186
-          </Typography>
-          <Typography
-            variant="subtitle1"
-            component="div"
-            fontWeight="bold"
-            color="primary.main"
-            sx={{ mb: 1 }}
-          >
-            *
+            $19
           </Typography>
           <Box sx={{ width: 5 }} />
           <Typography variant="body2" color="text.secondary">
-            / developer.
+            / dev / month.
```

Comment:
Should we think about adding a toggle button so that visitors see the price billed monthly or annually? Guess it would be easier to understand that we support both methods.